src/common/utils/reduxStoreUtilty.js

```diff
@@ -0,0 +1,12 @@
+import { createStore } from "redux";
+import { setDataReduxSate } from "../../common/store/commonReduxStoreReducer";
+
+export const store = createStore(setDataReduxSate);
```

Comment:
Do we really need to create a new store? Can't we just use the store that is already instantiated in common/store/createStore.js?

Reply:
And as for the reducer, we can combine it into the rootReducer in rootApp/rootReducer.js file.

Reply:
> Do we really need to create a new store? Can't we just use the store that is already instantiated in common/store/createStore.js?

@hithacker  We tried to use createStore.js .But we were not able to subscribe to the reducer.
Also we checked in application the for createStore.js file and we identified that there is reference for the same.

`import { store } from '../../common/store/createStore';
import { setDataReduxSate } from "../../common/store/commonReduxStoreReducer";

class ReduxStoreUtility {

  constructor(){
    store.subscribe(setDataReduxSate); // failing for subscribe ,as store Object is undefined.
  }

  storeDispachObservations = (type, value) => {
    store.dispatch({
      type: type,
      value: value
    });
  };
}
k
const reduxStoreUtility = new ReduxStoreUtility();
export default reduxStoreUtility;`

we are planning to go with reduxStoreUtilty.js file as we made it generic and will resolved the commits regarding naming conventions.
Please let us know your feedback.